### PR TITLE
feat(ISSUE-10): create message

### LINKED ISSUE #10: feat: Create endpoint that create and store messages into a chat room.
## Description
This endpoint should store the messages of the chat rooms and sent the `WebSocket` events the corresponding users.

## Acceptance criteria
- [ ] The endpoint should validate that the requesting user is related to the chat room.
- [ ] The endpoint should be protected by a bearer token.
- [ ] The endpoint should only accept `application/json` as `Content-Type`.
- [ ] The endpoint should return the information of the new message created.

### PR DESCRIPTION
**Pull request by JoseNoriegaa**
# Description
This pr adds an endpoint to create messages into a chat room.

# Changes
- Resolves #10 

# Unsolved tasks
- Websocket events will be added in another PR
- Authentication will be added in another PR

### CODE REVIEW

**Review by hecttoressau — DISMISSED** [review]
LGTM

File: packages/api-gateway/__tests__/src/drivers/http/routes/chats/create-message.test.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+// Mocks
+const mockMessageText = faker.lorem.sentence();
+const mockUserId = faker.database.mongodbObjectId();
+const mockChatId = faker.database.mongodbObjectId();
+
+const mockChat = {
+  _id: faker.database.mongodbObjectId(),
+  chatId: mockChatId,
+  text: mockMessageText,
+  createdBy: mockUserId,
```
Comment (hecttoressau) [inline packages/api-gateway/__tests__/src/drivers/http/routes/chats/create-message.test.js:17]: Don't you need deletedAt field?

**Review by hecttoressau — APPROVED** [review]
LGTM

File: packages/api-gateway/__tests__/src/drivers/http/routes/chats/create-message.test.js (hunk 1 above)
Comment (JoseNoriegaa) [reply packages/api-gateway/__tests__/src/drivers/http/routes/chats/create-message.test.js:17]: Yeah, we don't need it, I'll remove it in my next PR

**Pull request merged**